src/main/java/ai/rev/speechtotext/models/vocabulary/CustomVocabularyOptions.java

```diff
@@ -0,0 +1,93 @@
+package ai.rev.speechtotext.models.vocabulary;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class CustomVocabularyOptions {
```

Comment:
I'm torn here. That name sounds like you are submitting options. Since the `CustomVocabulary` is required the `Options` part sounds misleading.

What about something like `CustomVocabularyContents`? What we are submitting is the contents of the vocabulary...